src/common/POFields/FieldOrganization.js

```diff
@@ -34,9 +33,13 @@ const FieldOrganization = ({
   const [selectedOrganization, setSelectedOrganization] = useState({});
 
   useEffect(() => {
-    if (id && selectedOrganization.id !== id) {
-      mutator.fieldOrganizationOrg.GET()
-        .then(setSelectedOrganization);
+    if (id) {
+      if (selectedOrganization.id !== id) {
+        mutator.fieldOrganizationOrg.GET()
+          .then(setSelectedOrganization);
+      }
+    } else {
+      setSelectedOrganization({});
```

Comment:
this is required, since there was a bug when you change ORder template and new template has no organization assigned, so id of vendor is cleared, but name is displayed as previous vendor.